mypy/checker.py

```diff
@@ -107,6 +107,10 @@
         ('is_upper_bound', bool),  # False => precise type
     ])
 
+# Keeps track of partial types in a single scope
+PartialTypeScope = NamedTuple('PartialTypeMap', [('map', Dict[Var, Context]),
```

Comment:
Shouldn't `NAME` in `NAME = NamedTuple('NAME', ...)` always be the same in l.h.s. and r.h.s.?